bin/colors

```diff
@@ -13,22 +13,22 @@ extension Dictionary {
 let inPath = "Design/Colors.json"
 let outPath = "Library/Styles/Colors.swift"
 
-let data = NSData(contentsOfFile: inPath)!
-data.length
+let data = try! Data(contentsOf: URL(fileURLWithPath: inPath))
```

Comment:
curious, are you using the `try!` bang instead of `try?` for simplicity/it's not the app?